### PR TITLE
filter field in collection

### PR DESCRIPTION
**Pull request by keemcode**

### CODE REVIEW

**Comment by keemcode** [comment]
Sir, I need this feature on the project I'm working on. I have tried locally and it works.

**Comment by keemcode** [comment]
sksksksksksksks

**Comment by keemcode** [comment]
skskskskskss

**Comment by keemcode** [comment]
I wanted to ad the abstract test suite so I can do a some of integration testing and that will lead to a lot of dependencies and I don't know how comfortable you will be with that.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Here's a sample JSON query:
     	"{"field":"birthDate","op":"eq","data":"1959-09-30T00:00:00.000Z"}," +
     	"{"field":"parent.id","op":"eq","data":"2"}," +
     	"{"field":"creationDate","op":"eq","data":"01-31-1980 +5"}" +
```
Comment (kiamesdavies) [inline README.md]: there should be a comma here

**Review by kiamesdavies — CHANGES_REQUESTED** [review]
Nice one

**Review by kiamesdavies — APPROVED** [review]

**Comment by keemcode** [comment]
@kiamesdavies will a new version of the library be automatically released?

**Comment by keemcode** [comment]
Thank you very much boss. 🙌

**Pull request merged**